app/src/main/java/ch/epfl/polybazaar/widgets/NoConnectionForListingDialog.java

```diff
@@ -31,7 +31,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState){
         AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
         builder.setMessage("No Internet connection found")
                 .setTitle("Information")
-                .setPositiveButton("send as soon as connection is available", new DialogInterface.OnClickListener() {
+                .setPositiveButton("Send as soon as connection is available", new DialogInterface.OnClickListener() {
```

Comment:
Again I know this is not part of this PR, but we should remember to put these strings in xml

Reply:
Yes, done